src/main/java/com/o19s/es/explore/ExplorerQuery.java

```diff
@@ -90,12 +95,20 @@ public int hashCode() {
     }
 
     @Override
-    public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost)
-            throws IOException {
+    public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost) throws IOException {
         if (!LTRSettings.isLTRPluginEnabled()) {
             throw new IllegalStateException("LTR plugin is disabled. To enable, update ltr.plugin.enabled to true");
         }
 
+        try {
+            return _createWeight(searcher, scoreMode, boost);
+        } catch (Exception e) {
+            ltrStats.getStats().get(StatName.LTR_REQUEST_ERROR_COUNT.getName()).increment();
+            throw e;
+        }
+    }
+
+    private Weight _createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost) throws IOException {
```

Comment:
Why add `_` for this method ? Looks not consistent with other private method naming convention. How about removing the `_` ? For example, can change to `createWeightInternal`